keras/layers/normalization.py

```diff
@@ -57,16 +57,6 @@ def build(self):
         self.running_mean = K.zeros(input_shape)
         self.running_std = K.ones(input_shape)
 
-        # initialize self.updates: batch mean/std computation
-        X = self.get_input(train=True)
-        m = K.mean(X, axis=0)
-        std = K.mean(K.square(X - m) + self.epsilon, axis=0)
-        std = K.sqrt(std)
-        mean_update = self.momentum * self.running_mean + (1-self.momentum) * m
-        std_update = self.momentum * self.running_std + (1-self.momentum) * std
-        self.updates = [(self.running_mean, mean_update),
-                        (self.running_std, std_update)]
-
```

Comment:
These lines belong in `build`. Can we detail why you thought it was necessary to move them to `get_output`?